src/Slider.jsx

```diff
@@ -335,11 +348,15 @@ export default React.createClass({
       document.attachEvent('onmouseup', this.endSlide);
     }
 
-    return this.killEvent(ev);
```

Comment:
At this point there's no other return value for `startMouseSlide`, so now this is the same as `startTouchSlide`